shaping/wrapping.go

```diff
@@ -963,6 +968,14 @@ func (l *LineWrapper) postProcessLine(finalLine Line, done bool) (WrappedLine, b
 //
 // The returned line is only valid until the next call to
 // [*LineWrapper.Prepare] or [*LineWrapper.WrapParagraph].
+//
+// If the LineWrapper's [WrapConfig].TruncateAfterLines
+// is non-zero, the final line of text returned by successive calls to WrapNextLine
+// may be truncated. The quantity of runes truncated by wrapping the line is
+// returned in [WrappedLine].Truncated. If this is non-zero AND the LineWrapper's
+// [WrapConfig].Truncator is set, the final run of the truncated will will be a copy
+// of the Truncator with its Runes.Count set to the quantity of runes truncated during
```

Comment:
Almost, I still trip over "truncated" because `Runes.Count` is not impacted by the truncation/wrap - could "runes truncated" be "runes in the current run"?

Reply:
I've pushed what I hope is an additional clarification. Does it help?

Reply:
It does, but I still struggle a little. I see the words are all in here but the implication is somewhat unclear. What if 

> If this is non-zero AND the LineWrapper's [WrapConfig].Truncator is set, the final [Output] of the truncated line will be a copy of the Truncator with its Runes.Count set to the quantity of runes truncated during line wrapping.

Becomes:

> If this is non-zero AND the LIneWrapper's [WrapConfig].Truncator is set, the final line will be split into two [Output]s. The penultimate will represent the input runes that are still visible before truncation, and the final line will be a copy of the Truncator with its Runes.Count set to the quantity of runes truncated during line wrapping.

Reply:
Since any line can contain any number of input runs, we have to be careful about the wording, but I see what you're going for.

> If this is non-zero AND the LineWrapper's [WrapConfig].Truncator is set, the final line will end with an extra [Output]s. [Output]s before the final [Output] will represent the input runes that are still visible before truncation, and the final [Output] will be a copy of the Truncator with its Runes.Count set to the quantity of runes truncated during line wrapping.

I think this is accurate, while preserving your point.

Reply:
Perfect thanks :)